bundles/org.openhab.binding.freeathomesystem/src/main/java/org/openhab/binding/freeathomesystem/internal/handler/FreeAtHomeBridgeHandler.java

```diff
@@ -0,0 +1,819 @@
+
+            ret = false;
+        }
+
+        return ret;
+    }
+
+    /**
+     * Method to restart Http connection
+     *
+     * @author Andras Uhrin
+     *
+     */
+    public boolean restartHttpConnection() {
+        boolean ret = false;
+
+        ret = closeHttpConnection();
+
+        if (ret) {
+            ret = openHttpConnection();
+        }
+
+        return ret;
+    }
+
+    /**
+     * Method to connect the websocket session
+     *
+     * @author Andras Uhrin
+     *
+     */
+    @SuppressWarnings("null")
+    public boolean connectWebsocketSession() {
+        boolean ret = false;
+
+        URI uri = URI.create("ws://" + ipAddress + "/fhapi/v1/api/ws");
+
+        try {
+            // Start socket client
+            if ((websocketClient != null) && (socket != null)) {
+                websocketClient.setMaxTextMessageBufferSize(8 * 1024);
+                websocketClient.setMaxIdleTimeout(BRIDGE_WEBSOCKET_RECONNECT_DELAY * 60 * 1000);
+                websocketClient.start();
+                ClientUpgradeRequest request = new ClientUpgradeRequest();
+                request.setHeader("Authorization", authField);
+                request.setTimeout(BRIDGE_WEBSOCKET_RECONNECT_DELAY, TimeUnit.SECONDS);
+                websocketClient.connect(socket, uri, request);
+
+                logger.debug("Websocket connection to SysAP is OK");
+
+                ret = true;
+            } else {
+                ret = false;
+            }
+        } catch (Exception e) {
+            logger.error("Error by opening Websocket connection [{}]", e.getMessage());
+
+            if (websocketClient != null) {
+                try {
+                    websocketClient.stop();
+
+                    ret = false;
+                } catch (Exception e1) {
+                    logger.debug("Error by opening Websocket connection [{}]", e1.getMessage());
+
+                    ret = false;
+                }
+            } else {
+                ret = false;
+            }
+        }
+
+        return ret;
+    }
+
+    /**
+     * Method to close the websocket connection
+     *
+     * @author Andras Uhrin
+     *
+     */
+    @SuppressWarnings({ "deprecation", "null" })
+    public void closeWebSocketConnection() {
+        if (socketMonitor != null) {
+            socketMonitor.stop();
+        }
+
+        if (jettyThreadPool != null) {
+            try {
+                jettyThreadPool.stop();
+            } catch (Exception e1) {
+                logger.debug("Error by closing Websocket connection [{}]", e1.getMessage());
+            }
+        }
+
+        if (websocketClient != null) {
+            try {
+                websocketClient.stop();
+            } catch (Exception e2) {
+                logger.debug("Error by closing Websocket connection [{}]", e2.getMessage());
+            }
+        }
+    }
+
+    /**
+     * Method to open the websocket connection
+     *
+     * @author Andras Uhrin
```

Comment:
`@author` annotation only needed on class level.

Reply:
The unnecessary @author entries are removed